lib/temporal/activity/context.rb

```diff
@@ -56,6 +56,10 @@ def headers
         metadata.headers
       end
 
+      def log_tag
```

Comment:
This method definitely belongs on `Metadata` and not `Context`, this is quite clear from the fact that it only calls "metadata" in it

Reply:
This method is removed now.